examples/run/Makefile

```diff
@@ -84,7 +84,7 @@ $(TARGET)/$(ENCODED_ENV):
 		close(foo); printf("export %s_ENCODED=%s\n", a[1], b);}' > $(TARGET)/$(ENCODED_ENV); \
 
 $(TARGET)/$(SECRETS_YAML):
-	source $(TARGET)/$(ENCODED_ENV); \
+	. $(TARGET)/$(ENCODED_ENV); \
```

Comment:
aren't these equivalent?

Reply:
apparently dash (ubuntu/debian's bash replacement) doesn't have source, so it fails. sigh.